api/tools/src/main/java/org/pmiops/workbench/tools/ManageClusters.java

```diff
@@ -140,13 +186,23 @@ private static void deleteClusters(
   public CommandLineRunner run() {
     return (args) -> {
```

Comment:
aside: Do we not yet have an argument parser for Java like we do in Ruby?

Reply:
We do, but it hasn't been backfilled to these older CLIs yet. I've filed https://precisionmedicineinitiative.atlassian.net/browse/RW-4340 for that. This situation could also use some more thought - having a ruby layer of flag parsing feeding into a duplicate Java layer of flag parsing is a sad state of affairs.

Reply:
I'd be interested to chat. we're looking at the same issue with ops tooling.